src/Microsoft.DotNet.Build.Tasks/PackageFiles/init-tools.cmd

```diff
@@ -56,6 +56,24 @@ if not [%NET46_PUBLISH_ERROR_LEVEL%]==[0] (
 	exit /b %NET46_PUBLISH_ERROR_LEVEL%
 )
 
+set ILASM_PROJECTJSON=%BUILDTOOLS_PACKAGE_DIR%\ilasm\project.json
```

Comment:
>Right now, we are able to run the latest ilasm on our 1.1 runtime, but I don't think it's a safe thing to rely on in the future.

I agree it isn't a great thing to rely on but if it does work right now I think I would prefer we do that as opposed to adding another layer of restore and tool isolation. We have the same problem for all of our tools in BuildTools, as they have to agree on the version of the runtime that they can work on.